cmd/gentkn/main.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"time"
+
+	"github.com/ardanlabs/service/internal/platform/auth"
+	jwt "github.com/dgrijalva/jwt-go"
+	"github.com/google/uuid"
+	"github.com/kelseyhightower/envconfig"
+)
+
+func main() {
+
+	// Define and process configuration for entire tool.
+	var cfg struct {
+		KeyID string `envconfig:"KEY_ID" default:"default"`
+
+		Claims struct {
+			Audience string   `envconfig:"AUDIENCE"`
+			Subject  string   `envconfig:"SUBJECT"`
+			Roles    []string `envconfig:"ROLES"`
+		} `envconfig:"CLAIMS"`
+	}
+	if err := envconfig.Process("GENTKN", &cfg); err != nil {
```

Comment:
Why are we not using our internal flag package? Then you also get the command line options for free. If there are bugs let's fix them. I know I have one pending.

Reply:
I get that, I want to know why we are passing, non-flagged arguments?

Reply:
Seems to go against unix conventions to have required flags for cli tools

Reply:
That coupled with not really having a good default for that value, seems proper to use a non-flag arg